### DISCUSSION TITLE
Validation for CI/CD pipelines

### DISCUSSION THREAD

**Discussion by sam-bailey** [git-lfs/git-lfs#4625] [Ideas] [answered]
It would be great if there was a simple command to validate that all the file types in the repo that are tracked by LFS are pointers as expected.

This could be used in CI / CD pipelines to ensure that contributors are using git lfs correctly, and aren't accidentally pushing the large files to git. It would avoid cases where someone has forgotten to install git lfs on their machine and pushes files to git that should be on LFS.

Perhaps it already exists but I couldn't see it?

**Comment by sam-bailey** [comment, marked as answer]
Actually I've just seen this: https://github.com/git-lfs/git-lfs/pull/4525
Looks like`git lfs fsck` is what I was looking for

**Reply by bk2204** [reply]
Yes, this will be in 3.0.0, which we hope to release shortly.  It's not available in any released version yet.
